bench/bench.py

```diff
@@ -287,11 +289,11 @@ def decomm():
                            help='AWS account access key')
     tornado.options.define('secret_key', type=str, default='',
                            help='AWS account secret key')
-    tornado.options.define('ami', type=str, default='ami-48fd2120',
+    tornado.options.define('ami', type=str, default='ami-02df9ea15c1778c9c',
```

Comment:
This ami does not seem to exist in the default region, us-east-1 (and I also checked a couple of other common regions and could not find it). Can you find an ami from the default region?

```
$ aws --region us-east-1 ec2 describe-images --image-ids ami-48fd2120
{
    "Images": [
        {
            "Architecture": "x86_64",
            "CreationDate": "2014-08-16T21:40:48.000Z",
            "ImageId": "ami-48fd2120",
            "ImageLocation": "ubuntu-us-east-1/images/hvm-instance/ubuntu-trusty-14.04-amd64-server-20140816.manifest.xml",
            "ImageType": "machine",
            "Public": true,
            "OwnerId": "099720109477",
            "State": "available",
            "BlockDeviceMappings": [],
            "Hypervisor": "xen",
            "Name": "ubuntu/images/hvm-instance/ubuntu-trusty-14.04-amd64-server-20140816",
            "RootDeviceType": "instance-store",
            "SriovNetSupport": "simple",
            "VirtualizationType": "hvm"
        }
    ]
}
$ aws --region us-east-1 ec2 describe-images --image-ids ami-02df9ea15c1778c9c

An error occurred (InvalidAMIID.NotFound) when calling the DescribeImages operation: The image id '[ami-02df9ea15c1778c9c]' does not exist
```

(I guess we should update to ubuntu-16.04 or ubuntu-18.04, but you probably did that, I just can't tell because I can't find the ami ...)